### PR TITLE
[ADH-4828]: SSM. Add ability to start demo cluster locally 

### PR DESCRIPTION
**Pull request by VitekArkhipov**
- written scripts for deploying singlehost and multihost hadoop cluster with ssm in docker containers
- added instructions to README.md

### CODE REVIEW

File: supports/tools/docker/README.md [hunk 1]
```diff
@@ -1,8 +1,53 @@
...
+
+## Singlehost configuration
+
+* Hadoop + SSM in one container
+* SSM metastore as postgres container
+
+Command to build docker images in singlehost cluster mode (from project root dir)
+
+```shell
+build-image.sh singlehost
```
Comment (iamlapa) [inline supports/tools/docker/README.md]: build-image.sh -> build-images.sh

File: supports/tools/docker/README.md [hunk 2]
```diff
@@ -1,8 +1,53 @@
...
+```
+
+Command to start docker containers 
+
+```shell
+docker-compose -f ${project-dir}/supports/tools/docker/singlehost/docker-compose.yaml up -d
```
Comment (iamlapa) [inline supports/tools/docker/README.md]: The web interface of the SmartServer is not accessible.
image

```shell
root@4e7d8f4903de:/# curl http://localhost:8081/
{"timestamp":"2024-08-16T07:08:58.230+00:00","status":404,"error":"Not Found","path":"/"}
```

File: build-images.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+if [ -z "$CLUSTER_TYPE" ]
+then
+  CLUSTER_TYPE=multihost
+fi
+
+echo "=============================="
+echo "      Rebuild the project     "
+echo "=============================="
+mvn clean package -Pdist,web,hadoop-${HADOOP_PROFILE} -DskipTests
```
Comment (tigrulya-exe) [inline build-images.sh]: The new profile for building and copying compiled frontend bundle to `smart-web-server` resources is `web-ui` 
```suggestion
mvn clean package -Pdist,web-ui,hadoop-${HADOOP_PROFILE} -DskipTests
```

File: supports/tools/docker/README.md (hunk 2 above)
Comment (tigrulya-exe) [inline supports/tools/docker/README.md]: nit: `docker-compose` is [deprecated](https://docs.docker.com/compose/migrate/), we can use `docker compose` instead

File: supports/tools/docker/README.md (hunk 1 above)
Comment (VitekArkhipov) [reply supports/tools/docker/README.md]: fixed

File: supports/tools/docker/README.md (hunk 2 above)
Comment (VitekArkhipov) [reply supports/tools/docker/README.md]: how quickly the world is changing. fixed

File: build-images.sh (hunk 3 above)
Comment (VitekArkhipov) [reply build-images.sh]: fixed

File: supports/tools/docker/README.md (hunk 2 above)
Comment (VitekArkhipov) [reply supports/tools/docker/README.md]: fixed

**Review by tigrulya-exe — APPROVED** [review]

**Pull request merged**